### ISSUE TITLE
Classification: gradient boosting

### ISSUE THREAD

**Issue by hombit** [snad-space/flare-classifier#8]
Gradient boosting classifier, for example with one of catboost / xgboost or lightgbm. Please remember about reproducibility and robustness: use deterministic random numbers, and "safe" train-test splitter (see #3 for details)
